### PR TITLE
move health-check into development/production blocks

### PR DESCRIPTION
**Pull request by hugoesthere**
Resolves https://github.com/Originate/exosphere/issues/843

### CODE REVIEW

File: src/types/service_development_config.go [hunk 1]
```diff
@@ -2,6 +2,7 @@ package types
 
 // ServiceDevelopmentConfig represents development specific configuration for a service
 type ServiceDevelopmentConfig struct {
-	Scripts map[string]string `yaml:",omitempty"`
-	Port    string            `yaml:",omitempty"`
+	HealthCheck string            `yaml:"health-check,omitempty"`
```
Comment (charlierudolph) [inline src/types/service_development_config.go]: Is it worth it to have this here just for consistency? We aren't using it for anything at the moment. @alexdavid thoughts?

Comment (alexdavid) [reply src/types/service_development_config.go]: Probably YAGNI, at least in the near term. We can always add it if needed

Comment (charlierudolph) [reply src/types/service_development_config.go]: Alright @hugobho lets remove this.

**Review by charlierudolph — APPROVED** [review]
Good to merge after deleting it from the development config

**Pull request closed without merging**